src/cluster_legacy.c

```diff
@@ -370,7 +419,10 @@ int auxTlsPortPresent(clusterNode *n) {
 typedef struct {
     size_t totlen; /* Total length of this block including the message */
```

Comment:
This is the existing code but since this struct goes over the wire so we shouldn't use platform specific data types like `size_t`. I think `int` is big enough but looks like we have to use `int64_t` for backcompat
```suggestion
    int64_t totlen; /* Total length of this block including the message */
```

Reply:
never mind. I misread the struct name. this is fine.